en/docs/chapter_graph/graph.md

```diff
@@ -55,8 +55,8 @@ As shown in the figure below, let the adjacency matrix be $M$, and the list of v
 Adjacency matrices have the following characteristics.
 
 - A vertex cannot be connected to itself, so the elements on the main diagonal of the adjacency matrix are meaningless.
-- For undirected graphs, edges in both directions are equivalent, thus the adjacency matrix is symmetric about the main diagonal.
-- By replacing the elements of the adjacency matrix from $1$ and $0$ to weights, it can represent weighted graphs.
+- For undirected graphs, edges in both directions are equivalent, thus the adjacency matrix is symmetric with regard to the main diagonal.
+- By replacing the elements of the adjacency matrix from $1$ and $0$ to weight values, it can represent weighted graphs.
 
 When representing graphs with adjacency matrices, it is possible to directly access matrix elements to obtain edges, thus operations of addition, deletion, lookup, and modification are very efficient, all with a time complexity of $O(1)$. However, the space complexity of the matrix is $O(n^2)$, which consumes more memory.
```

Comment:
1/ grammar error. The punctuation after this sentence "it is possible to xx" should be period "."
2/ In this context, "resulting in" is more suitable. 
"Thus" is often used to show a logical conclusion or consequence based on the preceding information. It implies that the statement being made is a direct outcome or inference from the previous statement.
"Resulting in" is used to highlight the cause-and-effect relationship between two actions or events. It emphasizes that the first action or event leads to or brings about the second action or event.

How about updating to this to enhance the coherence of the sentence expression:  "When representing a graph using an adjacency matrix, we can directly access matrix elements to retrieve edges, resulting in efficient operations for insertion, deletion, retrieval, and modification with a time complexity of O(1). However, the space complexity of the matrix is O(n^2), requiring a significant amount of memory."